### ISSUE TITLE
Mismatch between data and data model

### ISSUE THREAD

**Issue by br1970** [IBMPredictiveAnalytics/TWCo_Forecast_Daily#1]
Stream fails to execute - error message: "Data and data model does not match"

Environment:
- Mac OS Sierra v10.12.4
- SPSS Modeler v18
- R v3.2.0
- SPSS_Modeler_R_Essentials_18.0_FP1_Mac.zip

Console output snippet:

`[1] "https://api.weather.com/v1/geocode/48.139311/11.580559/forecast/daily/3day.json?language=en-US&apiKey=*************************&units=e"
Error in FUN(newX[, i], ...) : could not find function "content"
Error:  IBM SPSS CF error:  The error code is '1012'  With message 'Data and data model does not match.'`

Entire console output attached:
[console output.pdf](https://github.com/IBMPredictiveAnalytics/TWCo_Forecast_Daily/files/930417/console.output.pdf)

**Comment by Wenpei** [comment]
@br1970 Can you paste rest api link to Browser.
https://api.weather.com/v1/geocode/48.139311/11.580559/forecast/daily/3day.json?language=en-US&apiKey=*************************&units=e
to check if key works fine?

**Comment by br1970** [comment]
@Wenpei I got it to work. First I had to install R 3.2.5 instead of 3.2.0. Still after that, running the stream would fail, saying some libraries were missing. I installed each one by hand:
- install.packages("dplyr")
- install.packages("lubridate")
- install.packages("urltools")
- install.packages("mefa")
After that, the stream worked pulling data from the TWC API.

I would recommend that the documentation on GitHub would be updated (for all TWC extensions) to clearly specify the R version to use, and possible missing libraries could be added as done above. Thanks.
